Python/Product/Common/Strings.resx

```diff
@@ -2256,4 +2256,7 @@ Expand selection to the full expression?</value>
     <value>Run Mypy</value>
     <comment>Mypy is a product name and should not be localized</comment>
   </data>
+  <data name="InstalledPtvsdOutdatedMessage" xml:space="preserve">
+    <value>ptvsd({0}) you are using is older than the version shipped with VS ptvsd({1}).</value>
```

Comment:
Can we make this:
`You have an old version of ptvsd installed in the current environment (version {0}), but version {1} is included with Visual Studio. Please upgrade or uninstall ptvsd to get the latest functionality`